docker-dev/docker-compose.yml

```diff
@@ -64,7 +64,7 @@ services:
         labels:
             - "traefik.backend=omscore"
             - "traefik.port=80"
-            - "traefik.frontend.rule=HostRegexp:{domain:[a-z]+}"
+            - "traefik.frontend.rule=HostRegexp:{domain:[a-z0-9.]+}"
```

Comment:
aha just saw the slack comment, I thought this was to add some port thingy (for which you'd have needed `:`)